exercises/concept/parsing-log-files/.docs/instructions.md

```diff
@@ -80,23 +80,25 @@ lp.RemoveEndOfLineText("[INF] end-of-line23033 Network Failure end-of-line27");
 // => "[INF]  Network Failure "
 ```
 
-## 5. List lines with extremely weak passwords so the guilty can be punished
+## 5. Return formatted lines with secret-code
```

Comment:
I think the new version is an improvement, but it does make things about secrets codes now instead of being about passwords.
Having looked into the exercise a bit more, I think we want users to use two things:

1. Character classes (`\S` and `\s`)
2. Use capture groups to extract text

The current implementation is just very unintuitive. I'm wondering if we should maybe completely rething the task. The current modification still looks quite a lot like the existing one, and I'm wondering whether that is more of a burden than a benefit.

Do you maybe have any other ideas? One idea I had was to tie this task into the first task, which is about identifying garbled log lines. What if the current task's goal was to un-garble a log line?

So `"Any old [ERR] text"` would become `"[ERR] Any old text"`. We could then allow for whitespace before or after the brackets to nudge students to the `\s` character class: `"Any old \t  [ERR] \t\t text"`  would become `"[ERR] Any old text"`